MAUI/Cartesian-Charts/StackedLine100.md

```diff
@@ -0,0 +1,448 @@
+---
+layout: post
+title: Stacked Line 100 Chart in .NET MAUI Chart control | Syncfusion
+description: Learn here all about stacked line 100 chart support in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Stacked Line 100 Chart in .NET MAUI Chart
+
+
+The stacked line 100 chart is a type of chart that displays multiple line series stacked on top of each other, to indicate the cumulative value of the data points at 100%.
+
+To render a stacked line 100 chart, create an instance of the [StackingLine100Series]() and add it to the [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_Series) collection property of the [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
```

Comment:
address the all stacked line md file comments for stacked 100 line md too

Reply:
modified stacked line 100 based on stacked line suggestions